src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+
+  max-width: $maxWidth;
+  padding: 10px;
+
+  * {
+    @include flexBlock;
+    align-items: center;
+
+    width: $blockWidth;
+    height: $blockHeight;
+    box-sizing: border-box;
+
+    background-color: $blockColor;
+    border: 1px solid black;
+
+    font-size: 30px;
+  }
```

Comment:
that's a very bad approach to give styles, because you literally give it to every html element. just give each day class `calendar__day`, and give styles for it